models/User.js

```diff
@@ -1,15 +1,63 @@
 const mongoose = require('mongoose');
-const { Schema } = mongoose;
+const crypto = require('crypto');
+const uuidv1 = require('uuid/v1');
```

Comment:
@dcr7978933k  What's the use case for this?

Reply:
Crypto is used here for encrypting the password and we can use it directly with Node.js so no need of npm install
and uuid is a simple and a faster way of creating a longer string ,generating unique id for hashing the password is one of the use case of it.